setup.py

```diff
@@ -16,19 +16,24 @@
 TORCH_MAX_VER = '1.7.1'
 CYTHON_MIN_VER = '0.29.20'
 INCLUDE_EXPERIMENTAL = environ.get('KAOLIN_INSTALL_EXPERIMENTAL') is not None
+IGNORE_TORCH_VER = environ.get('IGNORE_TORCH_VER') is not None
 
 missing_modules = []
 torch_spec = importlib.util.find_spec("torch")
 if torch_spec is None:
     warnings.warn("Couldn't find torch installed, so this will try to install it. "
                   "If the installation fails we recommend to first install it.")
-    missing_modules.append(f'torch>={TORCH_MIN_VER},<={TORCH_MAX_VER}')
+    if IGNORE_TORCH_VER:
+        missing_modules.append('torch')
+    else:
+        missing_modules.append(f'torch>={TORCH_MIN_VER},<={TORCH_MAX_VER}')
```

Comment:
If I understand this well, this will also ignore `PYTORCH_VERSION`.

Let's say in the future I want to test 1.8 and not 1.9 (assume 1.9 exists for the example), and I want to let `setup.py` installs it, that's the purpose of `PYTORCH_VERSION`, here I believe it won't work because you can't set `PYTORCH_VERSION == 1.8` without `IGNORE_TORCH_VER` and as soon as you set `IGNORE_TORCH_VER` it will try to install the latest version of torch (in this example 1.9), instead of 1.8.

I think the behavior we want is something like that:
1) `PYTORCH_VERSION` set between `TORCH_MIN_VER` and `TORCH_MAX_VER`: force (re)installation of pytorch at `PYTORCH_VERSION`
2) `PYTORCH_VERSION` set outside the supported range and `IGNORE_TORCH_VER` not set: Throw an error
3) `PYTORCH_VERSION` set outside the supported range and `IGNORE_TORCH_VER` set: Throw a gentle warning and force (re)installation of pytorch at `PYTORCH_VERSION`

PYTORCH_VERSION not set:
4) `IGNORE_TORCH_VER` not set, pytorch not installed: install pytorch at latest it can between `TORCH_MIN_VER` and `TORCH_MAX_VER`
5) `IGNORE_TORCH_VER` set, pytorch not installed: install pytorch at latest available
6) `IGNORE_TORCH_VER` not set, pytorch installed: check version of installed pytorch, if not supported: try to force reinstallation of pytorch at supported version with a gentle warning (current behavior: https://github.com/NVIDIAGameWorks/kaolin/blob/master/setup.py#L31)
7) `IGNORE_TORCH_VER` set, pytorch installed: check version of installed pytorch, if not supported: throw a gentle warning

I'm still thinking about a clean way to implement that.

Also let me Know if you need some help on testing.

Reply:
I can definitely implement this, although I wonder about the usefulness of the PYTORCH_VERSION env variable. It seems that if you want to force an install of a specific version of PyTorch and you know exactly which version ahead of time, you should install separately (e.g., `pip install torch==PYTORCH_VERSION` followed by installing kaolin). What do you think about removing that variable entirely? Removing it would greatly simplify the logic involved and remove some redundant cases (e.g., cases where there is a current version installed and a different version specified by PYTORCH_VERSION). Let me know what you decide.

Reply:
Sorry, got a bit busy with some other stuff. Pushed another commit that removes PYTORCH_VERSION entirely, add warnings for the cases described above, and adds documentation for the IGNORE_TORCH_VER env variable. Let me know if there's anything else I can do!

Reply:
@Caenorst just double-checking on this; would be great to integrate it!